src/lambdas/initiateAthenaQuery/handler.ts

```diff
@@ -22,6 +24,15 @@ export const handler = async (
 
   const zendeskId = retrieveZendeskIdFromEvent(event)
   appendZendeskIdToLogger(zendeskId)
+
+  if (zendeskId.startsWith('MR')) {
+    logger.warn('Manual query detected, no need to run athena query')
```

Comment:
This should be an `logger.info`, not warn, because this is a normal scenario you want to support